cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -590,6 +590,22 @@ can then be used in a `SET` clause:
 MATCH (n)-[r:REL]->(m) SET n=properties(r)
 ----
 
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+MATCH (a), (b), allShortestPaths((a)-[r]->(b)) RETURN b
+
+MATCH (a), (b), shortestPath((a)-[r]->(b)) RETURN b
+----
+a|
+`shortestPath` and `allShortestPaths` with fixed length relationships has been deprecated because of redundancy instead use a Match with a limit of 1 or:
```

Comment:
I think "redundancy" is a bit to vague if we already provide reason (which I think we do not usually do).

```suggestion
`shortestPath` and `allShortestPaths` without <<cypher-pattern-varlength, variable-length relationship>> have been deprecated because there is no additional information over using a simple `MATCH` with a `LIMIT` of 1 or:
```

or simply leave it out:

```suggestion
`shortestPath` and `allShortestPaths` without <<cypher-pattern-varlength, variable-length relationship>> have been deprecated. Instead, use a `MATCH` with a `LIMIT` of 1 or:
```